src/pages/Encounters/ReportBuilder/index.tsx

```
@@ -0,0 +1,93 @@
+                    <Button
+                      variant="outline"
+                      size="sm"
+                      className="w-full"
+                      asChild
+                    >
+                      <Link href={`/reportbuilder/${reportTemplate.id}`}>
+                        {t("edit_template")}
+                      </Link>
+                    </Button>
+                  )}
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)